parser_email.py

```diff
@@ -1347,8 +1347,21 @@ def _set_sdi(default_id, input_dict):
     return phantom.APP_SUCCESS
 
 
-def _create_dict_hash(input_dict):
+def _get_fips_enabled(self):
```

Comment:
Helpers and logics are taken from https://github.com/splunk-soar-connectors/gsgmail/pull/2/files

The team agrees to repeat the helpers here, as we don't have shared modules as of now. See further discussion on Slack thread.